.gitignore

```diff
@@ -47,3 +47,5 @@ third_party/googletest
 !third_party/googletest/googletest/cmake
 !third_party/googletest/googletest/include
 !third_party/googletest/googletest/src
+/maci64
```

Comment:
I would recommend removing these ... it causes confusion and the naming is user specific anyway.

Reply:
Done.